### PR TITLE
updated the model and index-test.js for 100% code coverage

### PR DESCRIPTION
**Pull request by ryandens**
Did it for myself just to mess around with bespoken, but if you want it included here it is

### CODE REVIEW

**Comment by ryandens** [comment]
I noticed there weren't many very good examples of good CI with alexa. I have a project on my own repo that I was working on (using aws lambda and python), but eventually realized that there isn't as many resources/focus on python in the alexa community as I had thought. If you want some help building this project up, let me know!

File: models/en-US.json [hunk 1]
```diff
@@ -6,15 +6,24 @@
...
+                  ]
                 },
                 {
                   "name": "AMAZON.HelpIntent",
                   "samples": []
                 },
                 {
                   "name": "AMAZON.StopIntent",
-                  "samples": []
+                  "samples": [
```
Comment (jkelvie) [inline models/en-US.json]: Why add these additional utterances? We add default samples for the builtin intents. You can see them here:
https://github.com/bespoken/virtual-alexa/blob/master/src/BuiltinUtterances.ts

It doesn't look like you use the custom utterances, so I would not include this. Or if you do include it, just include the ones that are custom (stop hello world, end, and end hello world).

The same applies for cancel.

File: .travis.yml [hunk 2]
```diff
@@ -1,3 +1,8 @@
 language: node_js
 node_js:
-  - "6"
\ No newline at end of file
+  - "6"
+install:
```
Comment (jkelvie) [inline .travis.yml]: It does this implicitly, but I like this change - I think it's a good idea to make this explicit as this is meant for beginners.

File: README.md [hunk 3]
```diff
@@ -1,8 +1,10 @@
-[![Build Status](https://travis-ci.org/bespoken/super-simple-unit-testing.svg?branch=master)](https://travis-ci.org/bespoken/super-simple-unit-testing)
-[![codecov](https://codecov.io/gh/bespoken/super-simple-unit-testing/branch/master/graph/badge.svg)](https://codecov.io/gh/bespoken/super-simple-unit-testing)
+[![Build Status](https://travis-ci.org/ryandens/super-simple-unit-testing.svg?branch=master)](https://travis-ci.org/ryandens/super-simple-unit-testing)
```
Comment (jkelvie) [inline README.md]: Can you put the changes to the unit tests and travis into their own branch/pull request?

**Review by jkelvie — CHANGES_REQUESTED** [review]
Hi Ryan, thanks a lot for putting together this PR. I like the idea of expanding the test coverage, but can you put that in its own PR without the changes to the badges and package.json paths?

Additionally, I'm interested to hear your thoughts on where you would like to go with this. Open a ticket or just send me an email (jpk@bespoken.io) and we can discuss.

**Review by jkelvie — APPROVED** [review]

**Pull request merged**